Doc/library/ast.rst

```diff
@@ -146,6 +146,77 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module. Body contains a list of module's statements.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('x = 1'), indent=4))
+        Module(
+            body=[
+                Assign(
+                    targets=[
+                        Name(id='x', ctx=Store())],
+                    value=Constant(value=1))],
+            type_ignores=[])
+
+
+.. class:: Expression(body)
+
+   A single Python expression.
+
+   .. doctest::
+
+        >>> print(ast.dump(ast.parse('123', mode='eval'), indent=4))
+        Expression(
+            body=Constant(value=123))
+
+
+.. class:: Interactive(body)
+
+   A single Python interactive statement, just like in :ref:`tut-interac`.
```

Comment:
Is "just" needed/appropriate here? It changes the meaning to say that this is "similar to" the statements parsed in interactive mode, rather than actually being the AST node use for such (isn't it?)

```suggestion
   A single Python interactive statement, like in :ref:`tut-interac`.
```